### PR TITLE
StudentFinanceCalculator: Remove 2018 dates, add 2020 dates and adhere to style guide

### PR DESCRIPTION
**Pull request by issyl0**
https://trello.com/c/qhBsbXHr/1608-add-student-finance-rates-for-2020-2021

### CODE REVIEW

File: lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb [hunk 1]
```diff
@@ -21,9 +21,5 @@
 <% end %>
 
 <% content_for :post_body do %>
-  If you're a part-time student applying for the 2018 to 2019 academic year, you'll need to know how many credits you'll study and how many credits a full-time student would study.
-
-  Ask your university or college if you're not sure.
```
Comment (issyl0) [inline lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb]: This and the sentence above weren't detailed in the spreadsheet, but didn't make sense now we've removed the 2018-19 dates. Should they instead be updated to reference more recent years, or to remove the "applying for ... academic year" portion?

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <% content_for :title do %>
-  Do any of the following apply? (you might get extra funding.)
+  Do any of the following apply? (You might get extra funding.)
```
Comment (issyl0) [inline lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb]: On this page, there's an `eg` on line 9 in the `"low-income"` section which doesn't match the styleguide. The spreadsheet didn't say whether this should be changed or not.

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <% content_for :title do %>
-  Do any of the following apply (you might get extra funding)?
+  Do any of the following apply (You might get extra funding)?
```
Comment (issyl0) [inline lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb]: Same, with the `eg` on line 7 instead.

**Comment by issyl0** [comment]
7 failing tests, but they're all on tests with `income => funding_amount` mappings (for example, https://github.com/alphagov/smart-answers/blob/master/test/unit/calculators/student_finance_calculator_test.rb#L150-L172 from `master`). There's no guidance in the spreadsheet about how we update these for more recent years, short of going through and testing the calculator locally manually with each of the dates and incomes.

File: lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb (hunk 1 above)
Comment (OrcaTom) [reply lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb]: Hey Issy, you're right. Good spot. Looks to me like this should have been removed last year as that's the case whatever year you're applying. Please update so sentence reads:

"If you’re a part-time student you’ll need to know how many credits you’ll study and how many credits a full-time student would study."

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb (hunk 2 above)
Comment (OrcaTom) [reply lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb]: Change any eg you find to "for example" please.

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb (hunk 3 above)
Comment (OrcaTom) [reply lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb]: Change any eg you find to "for example" please.

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb (hunk 2 above)
Comment (issyl0) [reply lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_uk_full_time_students_only.govspeak.erb]: Thanks. Done!

File: lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb (hunk 3 above)
Comment (issyl0) [reply lib/smart_answer_flows/student-finance-calculator/questions/do_any_of_the_following_apply_all_uk_students.govspeak.erb]: Thanks. Done!

File: lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb (hunk 1 above)
Comment (issyl0) [reply lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb]: Thanks. Done!

**Pull request merged**